layouts/shortcodes/feature-gate-description.html

```diff
@@ -0,0 +1,18 @@
+{{- $id := .Get "name" -}}
+{{- $featureGatesBundle := site.GetPage "page" "docs/reference/command-line-tools-reference/feature-gates" -}}
```

Comment:
All the feature gate descriptions are in one directory, whether the gate is removed or not. When we mark a feature gate as removed, I intend that we do _not_ move the file.

At the moment we change content/en/docs/reference/command-line-tools-reference/feature-gates-removed/index.md; maybe in the future there's a different approach, but we still shouldn't need to move the description file.